src/EnergyPlus/DualDuct.cc

```diff
@@ -93,15 +93,11 @@ namespace DualDuct {
     // To encapsulate the data and algorithms required to
     // manage the DualDuct Systems Simulation
 
-    // METHODOLOGY EMPLOYED:
-    // Needs description, as appropriate.
-
-    // Using/Aliasing
-    using namespace DataLoopNode;
-    using DataHVACGlobals::SmallAirVolFlow;
-    using DataHVACGlobals::SmallMassFlow;
-    using namespace DataSizing;
-    using namespace ScheduleManager;
```

Comment:
Got rid of many `using` statements in this PR.  Tried to do it to every file I touched where reasonable to do so.